apps/timer/app.js

```diff
@@ -19,7 +19,17 @@ module.exports = function (activity) {
   activity.on('create', () => {
     logger.log('on create')
     activity.keyboard.on('click', (event) => {
-      logger.log('on key event: ' + event.keyCode)
+      logger.log('on click: ' + event.keyCode)
+      activity.tts.stop()
```

Comment:
Are `click` and `dbclick` handling same logic?

Reply:
Please check bug description, both needs break current ringtone playing.

Reply:
Then I'd suggest a templated function generator for both cases.

Reply:
Please check if it's your wanted.